tests/settings/ConfigurationGatheringTest.py

```diff
@@ -21,14 +21,19 @@ class ConfigurationGatheringTest(unittest.TestCase):
     def setUp(self):
         self.log_printer = LogPrinter(NullPrinter())
 
+        # Needed so coala doesn't error out
+        self.min_args = ['-f', '*.java', '-b', 'JavaTestBear']
```

Comment:
IMO yes, I planned to use QuotesBear on the project to make it a standard but didn't dare to do it yet because of one annotationbear issue and all the conflicts it would lead to